setuptools/_distutils/spawn.py

```diff
@@ -2,20 +2,47 @@
 
 Provides the 'spawn()' function, a front-end to various platform-
 specific functions for launching another program in a sub-process.
-Also provides the 'find_executable()' to search the path for a given
-executable name.
 """
 
+from __future__ import annotations
+
 import os
+import platform
+import shutil
 import subprocess
 import sys
+import warnings
+
+from typing import Mapping
 
 from ._log import log
 from .debug import DEBUG
 from .errors import DistutilsExecError
 
 
-def spawn(cmd, search_path=1, verbose=0, dry_run=0, env=None):  # noqa: C901
+def _debug(cmd):
+    """
+    Render a subprocess command differently depending on DEBUG.
+    """
+    return cmd if DEBUG else cmd[0]
+
+
+def _inject_macos_ver(env: Mapping[str:str] | None) -> Mapping[str:str] | None:
+    if platform.system() != 'Darwin':
+        return env
+
+    from distutils.util import MACOSX_VERSION_VAR, get_macosx_target_ver
```

Comment:
The rest of the file uses relative imports. Maybe in the future distutils should use relative imports in this line too for consistency.

Reply:
Yes, good catch. I'll make the change upstream and incorporate here.

Reply:
Done in 4e5d97d.

Reply:
I also observed there are lots of cases of absolute imports of distutils, so more consistency is needed, but that's an exercise for later.